### ISSUE TITLE
Fix Lerna hoist warnings on `npm run bootstrap`

### ISSUE THREAD

**Issue by wkillerud** [SpareBank1/designsystem#6]
We have a handful of Lerna hoist warnings that we should fix. See the pasted log below and fix the issues so a `npm run bootstrap` produces no warnings.

```bash
lerna WARN EHOIST_ROOT_VERSION The repository root depends on babel-cli@^6.26.0, which differs from the more common babel-cli@^6.24.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-details-list-react" package depends on babel-cli@^6.24.1, which differs from the hoisted babel-cli@^6.26.0.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-buttons" package depends on nsp@^3.1.0, which differs from the hoisted nsp@^3.2.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-datepicker-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-decorators-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-details-list-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-dropdown-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-expandable-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-lists-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-message-box-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-spinner-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-system-message-react" package depends on react@^16.2.0, which differs from the hoisted react@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-datepicker-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-decorators-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-details-list-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-dropdown-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-expandable-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-lists-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-message-box-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-spinner-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-system-message-react" package depends on react-dom@^16.2.0, which differs from the hoisted react-dom@^16.1.1.
lerna WARN EHOIST_ROOT_VERSION The repository root depends on rimraf@^2.6.2, which differs from the more common rimraf@^2.6.1.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-core-react" package depends on rimraf@^2.6.1, which differs from the hoisted rimraf@^2.6.2.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-icons-react" package depends on rimraf@^2.6.1, which differs from the hoisted rimraf@^2.6.2.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-system-message-react" package depends on prop-types@*, which differs from the hoisted prop-types@^15.6.0.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-decorators-react" package depends on react-addons-test-utils@^15.6.2, which differs from the hoisted react-addons-test-utils@^15.1.0.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-dropdown-react" package depends on react-addons-test-utils@^15.0.1, which differs from the hoisted react-addons-test-utils@^15.1.0.
lerna WARN EHOIST_PKG_VERSION "@sb1/ffe-tables-react" package depends on sinon@^4.2.1, which differs from the hoisted sinon@^4.1.3.
```

**Comment by selbekk** [comment]
Was this issue fixed with [this commit](https://github.com/SpareBank1/designsystem/commit/42fad9f2dae51f250c07d44b78c9b5b0fee153d1), @kwltrs?

**Comment by kwltrs** [comment]
Fixed in 42fad9f2dae51f250c07d44b78c9b5b0fee153d1

**Comment by kwltrs** [comment]
Fixed in 42fad9f2dae51f250c07d44b78c9b5b0fee153d1
